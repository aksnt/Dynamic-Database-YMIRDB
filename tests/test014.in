SET a 1 2 3 4 5 6 7 8
SET b a a a a 
SET b 6 7 8 9 5 4 2 1 4 
SET d a b
SET c d a b 
LIST KEYS
DROP 1
SNAPSHOT
DROP 1
APPEND a b
GET a
BYE
