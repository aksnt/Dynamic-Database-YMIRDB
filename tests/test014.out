> ok

> ok

> ok

> ok

> ok

> c
d
b
a

> no such snapshot

> saved as snapshot 1

> ok

> ok

> [1 2 3 4 5 6 7 8 b]

> bye
